lib/aiuta_flutter.dart

```diff
@@ -147,13 +148,27 @@ class Aiuta {
             listeners.selectUploadedImage(action.uploadedImage);
             break;
           case DeleteUploadedImagesAction():
-            listeners.deleteUploadedImages(action.uploadedImages);
+            try {
+              listeners.deleteUploadedImages(action.uploadedImages);
+            } catch (e) {
+              // Notify native
+              AiutaPlatform.instance.notifyAboutError(
+                error: AiutaError.failedDeleteUploadedImages,
+              );
+            }
             break;
           case AddGeneratedImagesAction():
             listeners.addGeneratedImages(action.generatedImages);
             break;
           case DeleteGeneratedImagesAction():
-            listeners.deleteGeneratedImages(action.generatedImages);
+            try {
+              listeners.deleteGeneratedImages(action.generatedImages);
```

Comment:
await?

Reply:
Yep, added `_errorHandler` for such safe calls